dev/ledger.md

```diff
@@ -368,8 +368,9 @@ A key registration transaction additionally has the following fields:
 The cryptographic hash of the fields above is called the _transaction
 identifier_.  This is written as $\Hash(\Tx)$.
 
-A valid transaction can either be a _signed_ transaction or a _multi-signed_
-transaction.  This is determined by the _signature_ of a transaction:
+A valid transaction can either be a _signed_ transaction, a _multi-signed_
+transaction, or a _logic-signed_ transaction.
```

Comment:
At some point we'll want to describe the structure of the logicsig struct here (like https://github.com/algorandfoundation/specs/pull/6 does for multisig)

Reply:
added a section like the msig section